sources/packages/web/src/components/common/students/StudentApplications.vue

```diff
@@ -215,18 +188,23 @@ export default defineComponent({
 
     onMounted(reloadApplications);
 
-    // pagination sort event callback
-    const paginationAndSortEvent = async (event: any) => {
-      loading.value = true;
-      currentPage.value = event?.page;
-      currentPageLimit.value = event?.rows;
-      await getStudentApplications(
-        event.page,
-        event.rows,
-        event.sortField,
-        event.sortOrder,
-      );
-      loading.value = false;
+    const paginationAndSortEvent = async (event: DataTableOptions) => {
+      currentPagination.value.page = event.page;
+      currentPagination.value.pageLimit = event.itemsPerPage;
+      if (event.sortBy.length) {
+        const [sortBy] = event.sortBy;
+        currentPagination.value.sortField =
+          sortBy.key as StudentApplicationFields;
+        currentPagination.value.sortOrder =
+          sortBy.order === "desc"
+            ? DataTableSortOrder.DESC
+            : DataTableSortOrder.ASC;
+      } else {
+        // Sorting was removed, reset to default
+        currentPagination.value.sortField = DEFAULT_SORT_FIELD;
+        currentPagination.value.sortOrder = DataTableSortOrder.DESC;
```

Comment:
Current implementation as default order as ASC. Can you please set it to the same?

![image](https://github.com/user-attachments/assets/c69d91ea-08cf-41c0-b677-163e416cb012)

Otherwise cancelled applications are loading on the beginning by default. 

![image](https://github.com/user-attachments/assets/89dc87c8-5803-4e53-b676-0d0b72f78d64)